kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -189,6 +193,9 @@ def exec_foundry_kompile(
     spec_module = 'FOUNDRY-SPEC'
     foundry_definition_dir = foundry_out / 'kompiled'
     foundry_main_file = foundry_definition_dir / 'foundry.k'
```

Comment:
Unrelated to this PR: I think we should have a separate folder for the K specs.

Reply:
Perhaps. I don't think ti matters to much where it goes. But I do want to put it in the Foundry output directory, to keep it in snyc with the other artifacts there.